recipes/viromeqc/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.0.2" %}
+{% set name = "viromeqc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CarsonJM/viromeqc/releases/download/{{ version }}/viromeqc{{ version }}.tar.gz
+  sha256: e4da8944f6213a1a4a0dfaab3cf2661b4b54f53be8a37f06e3c088c660052189
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - git
+    - python 3.7
+  run:
+    - python 3.7
```

Comment:
Please remove the strict pinning, they should not be needed.

Reply:
@bgruening Strict pinning has been removed. I initially thought there were some version restrictions but found that wasn't the case after some testing this morning.